api/v1beta1/ironicconductor_types.go

```diff
@@ -125,6 +129,10 @@ type IronicConductorSpec struct {
 	// or 'json-rpc' to use JSON RPC transport. NOTE -> ironic-inspector
 	// requires oslo.messaging transport when not in standalone mode.
 	RPCTransport string `json:"rpcTransport"`
+
+	// +kubebuilder:validation:Optional
+	// keystoneVars - (Hidden) Internally used map of Keystone API endpoints
+	KeystoneVars map[string]string `json:"-"`
```

Comment:
Do we really need to store this? Lookup whenever it is required should be fast/cheap enough. I'm not certain but it may even be the more idiomatic k8s way.

Reply:
Ok, I see what you mean. I will re-work this and push new change.

Reply:
I put the KeystoneVars back in, base on conversation on slack.